### ISSUE TITLE
What is required to be considered for inclusion as an extension?

### ISSUE THREAD

**Issue by apiszcz** [ivanlonel/postgis-with-extensions#20]
Two extensions for consideration
vector
MobilityDB

**Comment by ivanlonel** [comment]
Hi @apiszcz.

I added pgvector in the letest commit.

Honestly, there isn't a rule written in stone for the requirements for an extension to make it in. This was a small personal project, aimed mainly at learning Docker while building an image that collected extensions useful for my personal needs, mostly PostGIS, oracle_fdw and sqlite_fdw, then adding a lot of other extensions that didn't require much more work than a simple `apt-get install`.

I try to stick with extensions which:
- don't increase too much the image size
- aren't too niche
- work well in all Postgres versions supported by this repo (currently 12 to 15)
- are easy to install (I might ignore this requirement if someone adds the extension in a working, tested pull request)

Regarding MobilityDB, I might be mistaken, but it doesn't seem to work in Postgres 15 (it keeps throwing `undefined symbol: "pg_atoi"`), it seems to increase image size considerably and I'm not sure how niche is it. Also, I personally couldn't get it working in Postgres 14 either, as I got `ERROR:  42723: operator <-> already exists` when trying to run CREATE EXTENSION.

For more specific needs, it's allowed and encouraged to either fork this repo or use this as a base image and make modifications downstream.

Thanks for the suggestions!
